graylog2-web-interface/src/views/components/QueryBar.jsx

```diff
@@ -44,7 +43,7 @@ const QueryBar = ({ children, queries, queryTitles, router, viewMetadata }) => {
   const selectQueryAndExecute = queryId => onSelectQuery(queryId);
   return (
     <QueryTabs queries={queries}
-               selectedQuery={activeQuery}
+               selectedQueryId={activeQuery}
```

Comment:
I renamed this prop, because I `selectedQuery` sounds like the attribute is holding the hole query. I think `selectedQueryId` is easier to understand. I would love to rename `activeQuery` to `activeQueryId`.. but that would be too much for this PR and may be you prefer the other name.